Curve-finance-suite/Unkill/src/agent.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+import { provideFunctionCallsDetectorHandler } from "forta-agent-tools";
+import { stablePoolInterface } from "./abi";
+
+const CURVE_POOL_OWNER = "0xeCb456EA5365865EbAb8a2661B0c503410e9B347";
+
+export const createFinding = (functionCallInfo: any): Finding => {
```

Comment:
There is no need for `{}` brackets on the arrow function since there is an immediate return
Removing the `{}` brackets would then make the `return` keyword redundant because the return is implicit when there are no curly brackets on an arrow function

Reply:
I prefer to leave this decision to my formatter so my code will always have the same style, but I agree we should create a common formatter configuration and everyone should use it.